node/src/actors/peers_manager/handlers.rs

```diff
@@ -14,6 +14,9 @@ impl Handler<AddPeers> for PeersManager {
     fn handle(&mut self, msg: AddPeers, _: &mut Context<Self>) -> Self::Result {
         // Insert address
         log::trace!("Adding the following peer addresses: {:?}", msg.addresses);
+        if msg.src_address == None {
```

Comment:
You can add a comment explaining why do we need this:

    // Remove peers added manually from the ice bucket to ensure that they are always added
    // The easiest way to check if a peer was added manually is by using msg.src_address,
    // which is set to None when using the addPeers JSON-RPC method and also when reading peers from config

Also, clippy prefers `if msg.src_address.is_none()`